cloudify-manager-worker/templates/before_hook.yaml

```diff
@@ -79,3 +80,6 @@ data:
         ln -s --verbose "$OKTA" /etc/cloudify/ssl/okta_certificate.pem
       fi
     fi
+  
+    echo "Set crontab for /var/log directory"
+    echo "* * * * * /usr/bin/rsync -a /var/log/cloudify /tmp/log" | crontab -u cfyuser -
```

Comment:
1. it is necessary to share files with datadog agents to be pushed by them to datadog cloud. I could not mount the /var/log/cloudify catalog directly to the shared volume because the cloudify do not want to run correctly then. 
2. @yuvlar Didn't we have a plan to set datadog logging as an option in helm chart?

Reply:
@bartoszkosciug The thought was to have the option in our helm chart to support a possible installation of dataDog, but in the means of - allowing to add annotations, volume mounts, etc.. in a general way, but that'll help define dataDog as well